backend/src/folder.ts

```diff
@@ -0,0 +1,127 @@
+import { APIGatewayEvent, Context, ProxyResult } from 'aws-lambda'
+const Folder = require('../model/Folder')
+const querystring = require('querystring')
+
+const mockUserid = 1
+
+const getFolder = async (event?: APIGatewayEvent): Promise<ProxyResult> => {
+    const params = event?.queryStringParameters
+
+    if (!params?.courseId) {
+        return {
+            statusCode: 400,
+            body: JSON.stringify({
+                message: 'Missing parameters',
+            }),
+        }
+    }
+
+    await Folder.sync()
+    return Folder.findAll({
```

Comment:
Using promises and async/await in the same function choose the async/await pattern